misc/python/materialize/cli/orchestratord.py

```diff
@@ -0,0 +1,331 @@
+import argparse
```

Comment:
Nice, thanks for putting that togther @doy-materialize!
Seems like this file is missing the Copyright and lint is failing.